ui/pages/institutional/custody/custody.tsx

```diff
@@ -593,10 +685,9 @@ const CustodyPage = () => {
               id="selectAllAccounts"
               data-testid={`select-all-accounts-selected-${isCheckBoxSelected}`}
               name="selectAllAccounts"
-              marginLeft={2}
-              value={{}}
+              value={''}
               onChange={(e) => setSelectAllAccounts(e)}
-              checked={isCheckBoxSelected}
+              checked={isCheckBoxSelected ?? undefined}
```

Comment:
nit: you can leave it as `isCheckBoxSelected` because 
`true ?? undefined => true`
`undefined ?? undefined => undefined`